code/modules/surgery/organ_manipulation.dm

```diff
@@ -27,14 +27,18 @@
 	implements = list(/obj/item/organ = 100, /obj/item/reagent_containers/food/snacks/organ = 0, /obj/item/organ_storage = 100, /obj/item/device/mmi = 100)
 	var/implements_extract = list(/obj/item/hemostat = 100, /obj/item/crowbar = 55)
 	var/implements_mend = list(/obj/item/cautery = 100, /obj/item/weldingtool = 70, /obj/item/lighter = 45, /obj/item/match = 20)
+	var/implements_heal = list(/obj/item/stack/medical/bruise_pack = 90, /obj/item/stack/medical/gauze = 55, /obj/item/stack/medical/gauze/improvised = 35)
+	var/implements_heal_robotic = list(/obj/item/stack/nanopaste = 100, /obj/item/screwdriver = 30)
```

Comment:
Screwdriver should probably be upped from 30 to 50. Or nanopaste would need to be provided in robotics. This incorrectly assumes that R&D actually does their job and/or supplies items to anyone outside of mining.

Reply:
Makes for an exciting Robotics visit!